app/helpers/practical_supports_helper.rb

```diff
@@ -47,4 +47,13 @@ def practical_support_guidance_link
     link_to t('patient.practical_support.guidance_link', fund: ActsAsTenant.current_tenant.name), url.to_s, target: '_blank'
     # "For guidance on practical support, view the #{link_content}."
   end
+
+  def practical_support_display_text(practical_support)
+    content = [practical_support.support_type]
+    content.push "#{t('common.from')} #{practical_support.source}"
+    content.push "#{t('common.for')} #{number_to_currency(practical_support.amount)}" if practical_support.amount.present?
+    content.push '(confirmed)' if practical_support.confirmed?
+    content.push "#{t('common.on_')} #{practical_support.support_date.display_date}" if practical_support.support_date.present?
+    content.join(' ')
+  end
```

Comment:
This is pretty hardcoded to English grammar, but its probably good enough for now?

The more robust alternative I can think of would be to encode distinct strings for each variation that take variables, allowing each locale to arrange them as it sees fit. See the `guidance_link` entry for a simpler example.

Reply:
Do we want to display the fulfilled or nah? If we're worried about space, I think (C) and (F) are sufficient for this quick-view.
![image](https://github.com/user-attachments/assets/a5315bdf-197c-420a-9043-d7b69cb4aa4c)

Reply:
On last thing - this ordering implies that the Confirmed is on the amount, rather than the whole Support. What do you think of moving it to the end?

Reply:
I think I'm in favor of the guidance_link strategy if this winds up needing to expand or gets awkward. I'd rather solidify it before we put the time into anything particularly robust, and the spanish is close enough I think with the `de`s and `por`s mostly being clean subs for the froms and fors. 

I think reordering a little bit is a good idea; I'm less worried about space though, I think this frees us up to a lot more by delegating updating out! As a general rule of thumb I don't think we need _every_ unit of data displayed and can iterate a little on that, but I'd believe that this should be rearranged at least a little bit.